google/cloud/forseti/notifier/notifiers/inventory_summary.py

```diff
@@ -129,24 +148,29 @@ def _send_email(self, summary_data, details_data):
 
         gsuite_dwd_status = self._get_gsuite_dwd_status(summary_data)
 
-        email_content = email_util.render_from_template(
+        email_content = BaseEmailConnector.render_from_template(
```

Comment:
Should not be leaking the base connector here.  Should be able to call this from the connector instance.

Reply:
Done.